test/spec/recipientLists.spec.js

```diff
@@ -1,175 +1,177 @@
-      recipientLists.all(function(err, data) {
-        expect(client.get.firstCall.args[0].uri).to.equal('recipient-lists');
-        done();
-      });
+    it('should call client get method with the appropriate uri', function() {
+      recipientLists.all()
+        .then(function() {
+          expect(client.get.firstCall.args[0].uri).to.equal('recipient-lists');
+        });
     });
   });
 
   describe('find Method', function() {
-    it('should call client get method with the appropriate uri', function(done) {
+    it('should call client get method with the appropriate uri', function() {
       var options = {
         id: 'test'
       };
-      recipientLists.find(options, function(err, data) {
-        expect(client.get.firstCall.args[0].uri).to.equal('recipient-lists/test');
-        done();
-      });
+      recipientLists.find(options)
+        .then(function() {
+          expect(client.get.firstCall.args[0].uri).to.equal('recipient-lists/test');
+        });
     });
 
-    it('should throw an error if id is missing', function(done) {
-      recipientLists.find(null, function(err) {
-        expect(err.message).to.equal('id is required');
-        expect(client.get).not.to.have.been.called;
-        done();
-      });
+    it('should throw an error if id is missing', function() {
+      return expect(recipientLists.find(null)).to.be.rejectedWith('id is required');
     });
 
-    it('should allow show_recipients to be set in options', function(done) {
+    it('should allow show_recipients to be set in options', function() {
       var options = {
         id: 'test',
         show_recipients: true
       };
 
-      recipientLists.find(options, function(err, data) {
-        expect(client.get.firstCall.args[0].qs).to.deep.equal({show_recipients: true});
-        done();
-      });
+      recipientLists.find(options)
+        .then(function() {
+          expect(client.get.firstCall.args[0].qs).to.deep.equal({show_recipients: true});
+        });
     });
   });
 
   describe('create Method', function() {
-    var test_list = [
-      {
-        address: {
-          email: 'test@test.com',
-          name: 'test'
-        }
-      }
-    ];
-
-    it('should call client post method with the appropriate uri', function(done) {
-      var options = {
-        recipients: test_list
+
+    it('should call client post method with the appropriate uri', function() {
+      var testList = {
+        id: 'test_list',
+        recipient: [
+          {
+            address: {
+              email: 'test@test.com',
+              name: 'test'
+            }
+          }
+        ]
       };
 
-      recipientLists.create(options, function(err, data) {
-        expect(client.post.firstCall.args[0].uri).to.equal('recipient-lists');
-        done();
-      });
+      recipientLists.create(testList)
+        .then(function() {
+          expect(client.post.firstCall.args[0].uri).to.equal('recipient-lists');
```

Comment:
we should also test if the body is correct (same as `testList`). Test's name could be renamed if done in this ticket.